app/pages/sections/consignee/ConsigneeExcisePage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.sections.consignee
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object ConsigneeExcisePage extends QuestionPage[String] {
+
+  override def path: JsPath = JsPath \ toString
```

Comment:
Should have `consignee` in the path. So that the answers are saved as part of a section object in Mongo.

i.e.: 
```suggestion
  override def path: JsPath = JsPath \ "consignee" \ toString
```